internal/envoy/v3/route.go

```diff
@@ -402,3 +411,123 @@ func containsMatch(s string) *envoy_route_v3.HeaderMatcher_SafeRegexMatch {
 		SafeRegexMatch: SafeRegexMatch(regex),
 	}
 }
+
+func CookieRewriteConfig(policies []dag.CookieRewritePolicy) *any.Any {
+	codeTemplate := `
```

Comment:
yeah thats a nice idea, I think we could do some refactors to enable unit testing and linting and embed a lua file rather than inline string

might take that as a future action item if we keep this Lua long term (and i don't get it implemented in Envoy natively) and so we could update the other place we have a Lua script